source/core/src/test/com/csse3200/game/inventory/items/FoodItemTest.java

```diff
@@ -0,0 +1,102 @@
+package com.csse3200.game.inventory.items;
+
+import com.csse3200.game.components.CombatStatsComponent;
+import com.csse3200.game.entities.Entity;
+import com.csse3200.game.services.GameTime;
+import com.csse3200.game.services.ServiceLocator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import com.csse3200.game.inventory.items.food.Foods.Apple;
+import com.csse3200.game.inventory.items.food.Foods.Candy;
+import com.csse3200.game.inventory.items.food.Foods.Meat;
+import com.csse3200.game.inventory.items.food.Foods.Carrot;
+import com.csse3200.game.inventory.items.food.Foods.ChickenLeg;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+public class FoodItemTest {
+    private Apple apple;
+    private Candy candy;
+    private Meat meat;
+    private Carrot carrot;
+    private ChickenLeg chickenLeg;
+    private CombatStatsComponent stat;
+    private FoodItemTest.TestablePLayer player1;
+    private GameTime gameTime;
+    private static final long DURATION = 120000;
+
+    private static class TestablePLayer extends ItemUsageContext {
+        public TestablePLayer(Entity entity) {
+            super(entity);
+        }
+    }
+
+    @BeforeEach
+    void setUp() {
+        int maxHealth = 200;
+        int maxHunger = 100;
+
+        stat = new CombatStatsComponent(100, 50,0,0,0,0, true);
+        player1 = new FoodItemTest.TestablePLayer(new Entity().addComponent(stat));
+        gameTime = Mockito.mock(GameTime.class);
+        stat.setHealth(0);
+        ServiceLocator.registerTimeSource(gameTime);
+
+        apple = new Apple(1);
```

Comment:
items do not need to be instantiated before EACH test, only in the tests in which they are used, as there is no overlap